ci/jenkins/test.sh

```diff
@@ -425,6 +433,164 @@ function deliver_antrea_windows {
     rm -f antrea-windows.tar.gz
 }
 
+function deliver_antrea_windows_containerd {
+    echo "====== Cleanup Antrea Installation ======"
+    clean_up_one_ns "antrea-test"
+    kubectl delete -f ${WORKDIR}/antrea-windows-containerd.yml --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/kube-proxy-windows-containerd.yml --ignore-not-found=true || true
+    kubectl delete daemonset antrea-agent -n kube-system --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/antrea.yml --ignore-not-found=true || true
+
+    echo "====== Building Antrea for the Following Commit ======"
+    export GO111MODULE=on
+    export GOPATH=${WORKDIR}/go
+    export GOROOT=/usr/local/go
+    export GOCACHE=${WORKSPACE}/../gocache
+    export PATH=${GOROOT}/bin:$PATH
+
+    git show --numstat
+    make clean
+    ${CLEAN_STALE_IMAGES_CONTAINERD}
+    chmod -R g-w build/images/ovs
+    chmod -R g-w build/images/base
+    DOCKER_REGISTRY="${DOCKER_REGISTRY}" ./hack/build-antrea-linux-all.sh --pull
+    if [[ "$TESTCASE" == "windows-networkpolicy-process" ]]; then
```

Comment:
I didn't find a valid test case named as "windows-networkpolicy-process" in line 56, is it legacy code? If not, how to differentiate this from containerd with docker runtime?

Reply:
Yes, we don't need to keep this part for windows CI now.